src/components/include/application_manager/state_controller.h

```diff
@@ -223,6 +223,14 @@ class StateController {
    * @param app pointer to application to be exited
    */
   virtual void ExitDefaultWindow(ApplicationSharedPtr app) = 0;
+
+  /**
+   * @brief Sets BACKGROUND or LIMITED hmi level to application
+   * depends on application type
+   * @param app Application to deactivate
+   */
+  virtual void DeactivateApp(ApplicationSharedPtr app,
```

Comment:
@AByzhynar missing description of window_id param

Reply:
@AKalinich-Luxoft Thank you. Done in 6467b21e3df1cb7c9fc032b91f45d27e042b0063